libraries/c_sdk/standard/mqtt/test/unit/iot_tests_mqtt_api.c

```diff
@@ -544,9 +544,9 @@ static int32_t transportSend( const NetworkContext_t * pNetworkContext,
 /**
  * @brief A transport send function that delays.
  */
-static size_t transportSendDelay( const NetworkContext_t * pSendContext,
-                                  const void * pMessage,
-                                  size_t messageLength )
+static int32_t transportSendDelay( const NetworkContext_t * pSendContext,
```

Comment:
After changing the return type, you now have an unsigned/signed conversion at line 563.  At a minimum, that should be marked with a typecast to avoid compiler/static analysis warnings.

Reply:
Done. Thanks